mass : 2.045
roll_time_constant  : 0.851609
roll_gain           : 1.024028
pitch_time_constant : 0.339568
pitch_gain          : 0.911249

drag_coefficients : [0.03, 0.03, 0]

q_position_x : 40
q_position_y : 40
q_position_z : 60

q_velocity_x : 40
q_velocity_y : 40
q_velocity_z : 60

q_attitude_pos : 20
q_attitude_vel : 10

r_command : [40, 40, 1]

roll_limit : 20
pitch_limit : 20
yaw_rate_limit : 3
thrust_min : 5
thrust_max : 25

k_yaw : 0.5

thrust_to_throttle : 0.666
throttle_limit : 0.95
